### PR TITLE
Editor page

### LINKED ISSUE #50: Change individual team page route to /teams/:id
Discussed here: https://github.com/developmentseed/osm-teams/pull/46#discussion_r308377111

Using `/teams/:id` is nice because it follows the REST pattern and makes it possible to edit the url down to `/teams` and get an expected result.

### PR DESCRIPTION
**Pull request by kamicut**
- Adds Formik for edit form
- Fixes session storage to have the user information in sessionStorage (might want to move this to state management later)
- Also Closes #50 

Note: 
There might be a sync error between token on the server and in the sessionStorage. This will be handled by "unauthorized" responses in the API, but we should have some kind of client side controller for messages and kicking the user back to login when the token is expired.

### CODE REVIEW

File: lib/teams-api.js [hunk 1]
```diff
@@ -23,3 +35,39 @@ export async function createTeam (data) {
...
+/**
+ * updateTeam
+ * Update a team
+ *
+ * @param id - Team id
+ * @param values - Basic details { bio, name, hashtag }
+ * @returns {Response}
+ */
+export async function updateTeam (id, values) {
+  return fetch(join(URL, id), {
```
Comment (sethvincent) [inline lib/teams-api.js]: `join` fails when `id` is a number because it is expecting a string.

Comment (kamicut) [reply lib/teams-api.js]: woops sorry about that. Fix incoming

Comment (sethvincent) [reply lib/teams-api.js]: @kamicut we could either do `String(id)` in the couple of places where it is necessary or revise the `URL` var and do something like: ```join(URL, `teams/${id}`)```

Comment (sethvincent) [reply lib/teams-api.js]: oh nvm you fixed it.

**Pull request merged**